frontend/src/app/shared/components/modals/preview-modal/hover-card-modal/hover-card.modal.sass

```diff
@@ -4,7 +4,8 @@
   position: absolute
   background-color: var(--body-background)
   z-index: 5000
-  min-width: 350px
-  box-shadow: var(--shadow-floating-large)
+  min-width: 500px
```

Comment:
Hi @HDinger 
According to specifications, it should have a fixed with of 500px, should it be width instead of min-width?